### DISCUSSION TITLE
cypher code syntax highlighting

### DISCUSSION THREAD

**Discussion by adriens** [community/community#32466] [Other Feature Feedback, Questions, & Ideas]
# :grey_question: About

Actually we use cypher in some Neo4J projects, but when we quote the code, we cannot get the same comfort than we have with other languages/codes (`java`, `json`, `yaml`, `SQL`,...) : 

```
MATCH (n:Application) 
WHERE n.api_backend_url is not null
RETURN n.name as nom, n.api_backend_url as url_aws;
```

# :dart: Feature request

Please add syntax highlighting so we can make our issues and docs better :pray:

**Comment by 007vasy** [comment]
https://github.com/github/linguist/pull/6027

I think this PR takes cares of your issue

**Reply by adriens** [reply]
:pray:

**Comment by strong-728** [comment]
you can use the vs code extentions.
Good luck.

**Reply by adriens** [reply]
:point_right:  The purpose is to have a proper render, right into issues, discussions, wikis,... :point_left:
